kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/kafkaservice/KafkaServiceReconciler.java

```diff
@@ -53,6 +77,18 @@ public List<EventSource<?, KafkaService>> prepareEventSources(EventSourceContext
 
     @Override
     public ErrorStatusUpdateControl<KafkaService> updateErrorStatus(KafkaService resource, Context<KafkaService> context, Exception e) {
-        return io.javaoperatorsdk.operator.api.reconciler.Reconciler.super.updateErrorStatus(resource, context, e);
+        var now = ZonedDateTime.ofInstant(clock.instant(), ZoneId.of("Z"));
```

Comment:
This is clunky, I'm almost tempted to say we should have our own clock wrapper to do this for us so can just say `clock.zonedInstant()` or something. Nothing to change in this PR however.

Reply:
Agree on both points.

Reply:
https://github.com/kroxylicious/kroxylicious/pull/1985